content/hr/blog/sample-letter-gdpr-access-request/index.md

```diff
@@ -8,36 +8,39 @@
 
-<p>[Molim vas da mi omogućite pristup osobnim podacima u strukturiranom, uobičajenom i strojno čitljivom formatu, kako je utvrđeno u članku 20. stavka 1. GDPR-a.]</p>
+<p>[Molim vas da moje osobne podatke koje sam vam dostavio/dostavila budu dostupni u strukturiranom, uobičajeno korištenom i strojno čitljivom formatu kako je navedeno u članku 20. stavka 1. GDPR-a.]</p>
 
-<p>Moj zahtjev izričito uključuje sve druge usluge i poduzeća za koje ste voditelj obrade, kako je definirano u članku 4. stavka 7. GDPR-a.</p>
+<p>Moj zahtjev izričito uključuje sve druge usluge i poduzeća za koje ste zaduženi kako je definirano člankom 4. stavka 7. GDPR-a.</p>
 
-<p>Kao što je utvrđeno u članku 12. stavka 3. GDPR-a, tražene podatke morate mi dostaviti bez odgađanja, a u svakom slučaju u roku od mjesec dana po primitku zahtjeva. Na osnovi članka 15. stavka 3. GDPR-a, na ovaj zahtjev morate odgovoriti bez naknade.</p>
+<p>Kao što je navedeno u članku 12. stavka 3. GDPR-a, morate mi dostaviti tražene podatke bez nepotrebnog odgađanja, a u svakom slučaju u roku od mjesec dana od primitka zahtjeva. Prema članku 15. (3) GDPR-a, morate odgovoriti na ovaj zahtjev bez ikakvih troškova za mene.</p>
 
-<p>Moji identifikacijski podaci:<br>
-<span class="blog-letter-fill-in">Ovdje navedeni podaci pomažu poduzećima u identifikaciji korisnika. Slobodno najprije upiši manje podataka – poduzeće će te kontaktirati, ako trebaju daljnje podatke.</span></p>
+<p>ukjučujem sljedeće podatke potrebne za identifikaciju:<br>
+<span class="blog-letter-fill-in">Ovdje upiši svoje identifikacijske podatke. To često uključuje podatke kao što su tvoje ime, datum rođenja, adresa, e-mail adresa itd.</span></p>
 
-<p>Ako ne odgovorite na moj zahtjev u navedenom roku, zadržavam si pravo poduzeti pravne korake protiv vas i podnijeti žalbu nadležnom nadzornom tijelu.</p>
+<p>Ukoliko ne odgovorite na moj zahtjev u navedenom roku, pridržavam si pravo podnijeti pravne korake protiv vas i podnijeti žalbu nadležnom nadzornom tijelu.</p>
 
-<p>Hvala unaprijed.</p>
+<p>Hvala vam unaprijed.</p>
 
-<p>S poštovanjem,<br>
-<span class="blog-letter-fill-in">Ime</span></p>
+<p>S poštovanjem</p>
+
+<p><span class="blog-letter-fill-in">Ovdje upiši svoje ime.</span></p>
 </div>
+
+<a href="/downloads/primjerak-pisma-gdpr-zahtjev-pristup-osobnipodaci.org.ott" class="button button-primary" style="margin-bottom: 10px;">Download LibreOffice or OpenOffice template (.ott)</a><br>
+<a href="/downloads/primjerak-pisma-gdpr-zahtjev-pristup-osobnipodaci.org.dotx" class="button button-secondary" style="margin-bottom: 10px;">Download Word template (.dotx)</a><br>
+<a href="/downloads/primjerak-pisma-gdpr-zahtjev-pristup-osobnipodaci.org.pdf" class="button button-secondary" style="margin-bottom: 10px;">Download PDF form (.pdf)</a><br>
+<a href="/downloads/primjerak-pisma-gdpr-zahtjev-pristup-osobnipodaci.org.txt" class="button button-secondary">Download plain text file (.txt)</a>
```

Comment:
Can you please provide translations for these strings?

* Download LibreOffice or OpenOffice template (.ott)
* Download Word template (.dotx)
* Download PDF form (.pdf)
* Download plain text file (.txt)